modules/swagger-codegen/src/main/resources/JavaSpring/beanValidation.mustache

```diff
@@ -1,5 +1,6 @@
 {{#required}}
   @NotNull
-{{/required}}
+{{/required}}{{#isContainer}}{{^isPrimitiveType}}{{^isEnum}}
```

Comment:
Can you explain the difference? Is there a reason not to include `@Valid` for any attribute on the bean?

I've been using it on all attributes in production for 3+ months now with no issues and validation has been enforced successfully.

So understand the difference and what attributes are not being validated and why would be helpful.

Reply:
@ebelanger I try to use {{#isContainer}}{{^isPrimitiveType}}{{^isEnum}} in the Resteasy beanValidation.mustache template but it doesn't seem to be working (No @Valid adds on the string type of values). Add you saying array is not an object so that @Valid can't validate it?

Reply:
@magictortoise like mentionned in a previous comment, validating an array with @Valid causes a ClassCastException in Hibernate Validation when it tries to cast it to an Object.

Reply:
@ebelanger Do you know where I can learn to use tags like {{#isContainer}} and how can I debug if it is not working?